apps/ansys_mechanical/readme.md

```diff
@@ -0,0 +1,70 @@
+## Install and run Ansys Mechanical Benchmarks
+
+## Prerequisites
+
+Cluster is built with the desired configuration for networking, storage, compute etc. You can see the tutorial or examples folder in this repo for how to set this up.
+
+Recommended that you start with the cfd_workflow tutorial for the cluster setup since you need extra disk space for the install and running of the benchmarks.
+
+Dependencies for binary version:
+
+* v19.*
+
+## Installation
+
+NOTE: Update the path to the fluent installer tar file in $azhpc_dir/apps/fluent/install_fluent.sh
+
+First copy the apps directory to the cluster.  The `azhpc-scp` can be used to do this:
+
+```
+azhpc-scp -u hpcuser -r $azhpc_dir/apps hpcuser@headnode:.
+```
+
+### Install Prerequisite Software
+```
+azhpc-run -u hpcuser -n "headnode compute" ~/apps/ansys_mechanical/scripts/add_reqs.sh
+```
+
+### Install Ansys Mechanical
+```
+azhpc-run -u hpcuser -n headnode ~/apps/ansys_mechanical/install_mechanical.sh \<tar file name\> \<URL to tar file of the installer\>
```

Comment:
@edwardsp - how does azhpc-run handle this? can you pass args to the script here?